tinywl/tinywl.c

```diff
@@ -846,7 +846,7 @@ int main(int argc, char *argv[]) {
 	 * backend uses the renderer, for example, to fall back to software cursors
 	 * if the backend does not support hardware cursors (some older GPUs
 	 * don't). */
-	server.backend = wlr_backend_autocreate(server.wl_display, NULL);
+	server.backend = wlr_backend_autocreate(server.wl_display);
```

Comment:
Can we remove the comment "The NULL argument here optionally allows you […]"?

Reply:
Oh sure, I squashed the fix into the first tinywl commit.